src/crystal.f90

```diff
@@ -26,14 +27,6 @@ module crystal_module
   private
   public crystal, calculate_wavevectors_full
 
-  !Captain's log. August 11, 2023.
-  !Ideally these should be immutables defined once and for all
-  !in params.f90. Also, these should be stored in a dictionary
-  !but I'm still shopping for a suitable one.
-  character(len = 3), allocatable :: isotope_element(:)
-  real(r64), allocatable :: isotope_mass(:)
-  real(r64), allocatable :: isotope_abundance(:)
-  
   type crystal
```

Comment:
Something must have gone wrong in the replacement of the older periodic table and how that was used. This breaks the regression test. See output of the GitHub CI.

Reply:
It will be useful to have a unit test before replacing the old periodic table. One possibility: The older table can be used to randomly pick a few elements and verify against the new dictionary style periodic table.

Reply:
I wrote a small test for the periodic table comparing the isotope data (mass/abundance) with the new and old versions. No differences were detected. FYI, I decided to check all data points rather than random ones.